Documentation/getting-started.md

```diff
@@ -0,0 +1,128 @@
+---
+weight: 201
+toc: true
+title: Getting Started
+menu:
+    docs:
+        parent: user-guides
+lead: ""
+images: []
+draft: false
+description: Getting started page for Platform Guide
+---
+
+This guide assumes you have a basic understanding of the Prometheus Operator. If you are new to it, please start with the [Introduction](introduction.md) page before proceeding. This guide will walk you through deploying Prometheus and Alertmanager instances.
+
+## Deploying Prometheus
+
+To deploy a Prometheus instance, you must first create the [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/authorization/) rules for the Prometheus service account if RBAC authorization is enabled on your cluster.
```

Comment:
I'm not aware that RBAC can be disabled 🤔 Is that really possible?

Reply:
Interesting, let's double check!

Reply:
This is the only blocker for now. Let's investigate if RBAC can really be disabled and if its not possible, let's rephrase this part